File: 1ba/PRG.H2503/golang/block01/aufgabe4.go
package main

import "fmt"

func main() {
	var guthaben float64
	var zinsSatz float64
	var guthabenMitZinsen float64

	fmt.Print("Geben Sie das Guthaben ein: ")
	fmt.Scanln(&guthaben)
	fmt.Print("Geben Sie den Zinssatz ein: ")
	fmt.Scanln(&zinsSatz)

	guthabenMitZinsen = guthaben * (1 + zinsSatz / 100)
	fmt.Println("Guthaben mit Zinsen:", guthabenMitZinsen)
}

File: 1.BA/PRG.H2503/block01/aufgaben/aufgabe4.java
import java.util.Scanner;
public class Aufgabe4 {
    public static void main(String[] args) {
        // Variablen
        double guthaben;
        double zinsSatz = 1.5;
        double guthabenMitZinsen;
        // Eingabe
        Scanner sc = new Scanner(System.in);
        System.out.print ("Geben Sie ein Guthaben ein:");
        guthaben = sc.nextDouble();
        // Verarbeitung
        guthabenMitZinsen = guthaben * (1 + zinsSatz / 100);
        // Ausgabe
        System.out.println("Guthaben nach einem Jahr: " + guthabenMitZinsen);
    }
}